packages/stashjs/src/auth/profile-store.ts

```diff
@@ -269,13 +271,13 @@ function sanitiseProfileName(profileName: string): string {
 
 /**
  * An implementation of ConfigStore that wraps all operations in an exclusive
- * lock (POSIX file lock).
+ * lock (using file locking - portable across Windows/Linux/macOS).
```

Comment:
Is this change coming later or has it already been implemented?

Reply:
Turns out that Windows was already supported by the `lockfile` package.

Reply:
Nice! Hopefully Rust has a similar package.